bundles/org.openhab.ui/web/src/js/store/index.js

```diff
@@ -23,7 +23,15 @@ const store = new Vuex.Store({
     developerSidebar: false
   },
   getters: {
-    apiEndpoint: (state) => (type) => (!state.apiEndpoints) ? null : state.apiEndpoints.find((e) => e.type === type)
+    apiEndpoint: (state) => (type) => (!state.apiEndpoints) ? null : state.apiEndpoints.find((e) => e.type === type),
+    locale: (state, getters) => {
+      const segments = state.locale?.split('_') || []
```

Comment:
Could you avoid the use of `?.` - I'm not strictly against starting using it at some point, but since it's the first and only occurrence, let's stick with old-school for a little while for consistency in the codebase :)

`const segments = (state.locale) ? state.locale.split('_') : []`

Reply:
Sounds good. Change is on the way. Should I create a separate PR and update the ESLint rule accordingly ?

Reply:
>  Should I create a separate PR and update the ESLint rules accordingly ?

Oh, sure!
I'll merge the rest tomorrow, need to check if anything breaks :)